test-clients/src/main/java/com/hedera/services/legacy/smartcontract/OCTokenIT.java

```diff
@@ -57,6 +57,7 @@
 import java.math.BigInteger;
```

Comment:
I converted this test case to EET based OCTokenSpec test suite. I think this test case doesn't need to be in part of the aggregated test any more.